manifests/n/NSIS/NSIS/3.07/NSIS.NSIS.yaml

```diff
@@ -0,0 +1,24 @@
+PackageIdentifier: NSIS.NSIS
+PackageVersion: 3.07
+PackageName: Nullsoft Install System
+Publisher: Nullsoft and Contributors
+PackageUrl: https://nsis.sourceforge.io
+ShortDescription: NSIS (Nullsoft Scriptable Install System) is a professional open source system to create Windows installers. It is designed to be as small and flexible as possible and is therefore very suitable for internet distribution.
+License: zlib/libpng, bzip2, CPLv1
+LicenseUrl: https://nsis.sourceforge.io/License
+Moniker: nsis
+Tags: [setup, installer]
+FileExtensions: [nsi, nsh]
+
+Scope: machine
+InstallModes: [interactive, silent]
+MinimumOSVersion: 4.0.1381.0
+Installers:
+- Architecture: x86
+  InstallerUrl: https://sourceforge.net/projects/nsis/files/NSIS%203/3.07/nsis-3.07-setup.exe/download
+  InstallerSha256: 82dbb5c52755e46477039bc9c4b13831b03fdcfd6f54a58b59b91802145383cf
+  InstallerType: nullsoft
```

Comment:
```suggestion
Moniker: nsis
Tags: 
- setup
- installer
FileExtensions: 
- nsi
- nsh
InstallModes: 
- interactive
- silent
MinimumOSVersion: 4.0.1381.0
Installers:
- Architecture: x86
  InstallerUrl: https://sourceforge.net/projects/nsis/files/NSIS%203/3.07/nsis-3.07-setup.exe/download
  InstallerSha256: 82dbb5c52755e46477039bc9c4b13831b03fdcfd6f54a58b59b91802145383cf
  InstallerType: nullsoft
  Scope: machine
```

Reply:
Why? [ ] is valid YAML syntax and passes the validator

`Scope:` is allowed in the root and all installers will support the same scope.

Reply:
The manifests was absolutely correct, but to

Reply:
The manifests were absolutely correct. Just to make it look like other manifests, I suggested changes.